src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java

```diff
@@ -467,19 +467,25 @@ public void onGuiScreenDrawPre(GuiScreenEvent.DrawScreenEvent.Pre event) {
 			"Spirit Leap",
 			"Revive A Teammate",
 			"Click in order!",
-			"What starts with",
-			"Select all the",
 			"Click the button on time!",
 			"Correct all the panes!",
 			"Change all to same color!"
 		};
 	private static final Set<String> dungeonMenuSet = new HashSet<>(Arrays.asList(dungeonMenus));
 
+	private boolean isInDungeonMenu(String chestName) {
+		if (!SBInfo.getInstance().isInDungeon) {
+			return false;
+		}
+		return dungeonMenuSet.contains(chestName) ||
+			chestName.startsWith("What starts with") ||
```

Comment:
maybe have a starts with list and an equals list?

Reply:
Looks also ugly in code